[rum] added an info panel clarifying support of e2e tracing (#10478)
DataDog/documentation: content/en/real_user_monitoring/connect_rum_and_traces/_index.md

**Note**: `allowedTracingOrigins` accepts Javascript String and RegExp that should match the origins called by your browser application, with origin defined as: `<scheme> "://" <hostname> [ ":" <port> ]`

> <div class="alert alert-info">The browser SDK must be initialized in order to link browser requests to backend traces. End-to-end tracing of the initial HTML document, and early browser requests, is not currently supported.</div>

Reviewer: ```suggestion
<div class="alert alert-info">To link browser requests to backend traces, initialize the browser SDK. End-to-end tracing of the initial HTML document and early browser requests is not supported.</div>
```

Author: The 1st sentence loses the intended meaning: end-to-end tracing will only work for requests fired after the browser SDK is initialized.

Reviewer: Let's say: 
```suggestion
<div class="alert alert-info">End-to-end tracing is available for requests fired after the browser SDK is initialized. End-to-end tracing of the initial HTML document, and early browser requests, is not supported.</div>
```
